pandera/errors.py

```diff
@@ -166,6 +166,9 @@ def _message(self, error_counts, schema_errors):
             msg += f"- {k}: {v}\n"
 
         def agg_failure_cases(df):
+            # TODO(arne): hack to support unhashable types, proper solution that only
```

Comment:
I get a failure from pylint on this line, I can't use TODOs? What should I do instead, write a ticket?